<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PadSettingsDialog</class>
 <widget class="QDialog" name="PadSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="padLayoutView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="padSelectionSpin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>57</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>48</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pad Selection</string>
   </property>
  </widget>
  <widget class="QComboBox" name="padcolor_on">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>80</y>
     <width>72</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="LEDs Off">Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Amber">Amber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Red">Red</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="padNoteSpin">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>80</y>
     <width>57</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Midi Value for Note, Controller Number for Control. 1 - 128, 0 to disable."/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="padChannelSpin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>57</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="padMode_note">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>83</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="padMode_control">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>83</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="padcolor_off">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>80</y>
     <width>72</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="LEDs Off">Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Amber">Amber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Red">Red</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="padcolor_inactive">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>72</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="LEDs Off">Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Amber">Amber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="LED Red">Red</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inactive</string>
   </property>
  </widget>
  <widget class="QLabel" name="debugLbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PadSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PadSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
